app/models/SupportFrontendSwitches.scala

```diff
@@ -45,7 +45,9 @@ case class SupportFrontendSwitches(
   enableDigitalSubGifting: SwitchState,
   useDotcomContactPage: SwitchState,
   enableRecaptchaBackend: SwitchState,
-  enableRecaptchaFrontend: SwitchState
+  enableRecaptchaFrontend: SwitchState,
+  enableContributionsCampaign: SwitchState = Off,
+  forceContributionsCampaign: SwitchState = Off,
```

Comment:
these switches were previously in the `experiments` switches, and this PR is formalising them in the model